src/components/clusterprovider/minikube/minikube.ts

```diff
@@ -84,7 +84,10 @@ class MinikubeImpl implements Minikube {
 }
 
 async function minikubeUpgradeAvailable(context: Context): Promise<void> {
-    if (!await checkPresent(context, CheckPresentMode.Silent)) {
+
+    const checkForUpgrade = vscode.workspace.getConfiguration().get("vsminikube.checkForUpgrade");
```

Comment:
I've been trying to contain configuration checks and keys within the `components/config/config.ts` file - would you mind encapsulating this there?